css/components/navigation.scss

```diff
@@ -1,33 +1,100 @@
 // Navbar styles
+
 .navbar {
-  width: 85%;
+  width: 100%;
   height: auto;
-  line-height: 1.2;
   margin: 0 auto;
   display: flex;
```

Comment:
I'd recommend against using flex-box for now 😞 , it's not a finished standard across even the most up-to-date browsers and anything older will just cry. I think it messes a lot with animations right now too. The only time to use it would be if you really know you need it.

You can accomplish what you want by using `display: inline-block` and `float: left/right` with some more markup (HTML/JSX). Take a look [here](https://github.com/jubearsun/innovative-design/blob/master/css/components/nav.scss).

Since you guys aren't doing split left/right in the nav and just center links, you could probably forgo the floats and just do `display: inline-block` on all your `.navbar__link--wrappers` and add `margin-left` and `margin-right` to space them. And to center the link-wrappers you can just do `text-align: center` on `.navbar`